cmd/mothership/mothership/start/auditlog.go

```diff
@@ -59,17 +63,28 @@ func NewLoggerWithFile(logFile string) (*zap.Logger, error) {
 	), err
 }
 
-func NewAuditLoggerMiddelware(l *zap.Logger) func(http.Handler) http.Handler {
+func NewAuditLoggerMiddelware(l *zap.Logger, o *Options) func(http.Handler) http.Handler {
```

Comment:
Would recommend to make it a private function as it shouldn't be used outside of this pkg.